src/python/tests/test_statistics_history.py

```diff
@@ -0,0 +1,276 @@
+
+import asyncio
+import aiopg.sa
+import datetime
+import pytest
+import sqlalchemy as sa
+
+__author__ = "Ori Shadmon"
+__copyright__ = "Copyright (c) 2017 OSIsoft, LLC"
+__license__ = "Apache 2.0"
+__version__ = "${VERSION}"
+
+
+_connection_string = "dbname='foglamp'"
+
+async def _delete_from_statistics():
+    """DELETE data from table"""
+    try:
+        async with aiopg.sa.create_engine(_connection_string) as engine:
+            async with engine.acquire() as conn:
+                await conn.execute('DELETE FROM foglamp.statistics')
+                await conn.execute('DELETE FROM foglamp.statistics_history')
+    except Exception:
+        print('DELETE failed')
+        raise
+
+async def _insert_init_data()->dict:
+    """
+    Insert initial data into statistics table
+    :return:
+        Dictionary with key being foglamp.statistics.key and value being foglamp.statistics.value
+    """
+    info = {'READINGS': 'The number of readings received by FogLAMP since startup',
+            'BUFFERED': 'The number of readings currently in the FogLAMP buffer',
+            'SENT': 'The number of readings sent to the historian',
+            'UNSENT': 'The number of readings filtered out in the send process',
+            'PURGED': 'The number of readings removed from the buffer by the purge process',
+            'UNSNPURGED': 'The number of readings that were purged from the buffer before being sent',
+            'DISCARDED': 'The number of readings discarded at the input side by FogLAMP, i.e. discarded ' +
+                         'before being placed in the buffer. This may be due to some error in the readings' +
+                         'themselves.'}
+    values = {}
+    for key in info.keys():
+        stmt = _statistics_tbl.insert().values(key=key, description=info[key], value=0,  previous_value=0)
+        try:
+            async with aiopg.sa.create_engine(_connection_string) as engine:
+                async with engine.acquire() as conn:
+                    await conn.execute(stmt)
+        except Exception:
+            print('INSERT failed')
+            raise
+        values[key] = 0
+    return values
+
+
+async def _update_statistics_value(values: dict={None:None})->dict:
+    """
+    Update values in statistics table
+    :param values:
+        values: "hard-coded" in dictionary correspond to key/value in foglamp.statistics
+    :return:
+        The updated values dictionary
+    """
+    for key in values.keys():
+        stmt = sa.update(_statistics_tbl).values(value=values[key]+1).where(_statistics_tbl.c.key == key)
+        try:
+            async with aiopg.sa.create_engine(_connection_string) as engine:
+                async with engine.acquire() as conn:
+                    await conn.execute(stmt)
+        except Exception:
+            print('Updated failed: %s' % stmt)
+            raise
+        values[key] =+ 1
+    return values
+
+
+@pytest.mark.asyncio
+async def test_init_data():
```

Comment:
Please add below annotations for all test cases in a file.
```
@pytest.allure.feature("unit")
@pytest.allure.story("statistics history")
```

Also, you may test and generate allure report on local itself after make changes. Please have a look on `docs->build-tools.rst`